### PR TITLE
fix: add confirmation for create technical screening button

### PR DESCRIPTION
**Pull request by valerydluski**
**🟢 Add `deploy` label if you want to deploy this Pull Request to staging environment**

#### 🧑‍⚖️ Pull Request Naming Convention

- Title should follow [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- Do not put issue id in title
- Do not put WIP in title. Use [Draft PR](https://github.blog/2019-02-14-introducing-draft-pull-requests/) functionality
- Consider to add `area:*` label(s)

* [x] I followed naming convention rules

---

#### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Performance optimization
- [ ] Refactoring
- [ ] Test Case
- [ ] Documentation update
- [ ] Other

#### 🔗 Related issue link

[issue](https://github.com/rolling-scopes/rsschool-app/issues/1965)

#### 💡 Background and solution

image

#### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Database migration is added or not needed
- [x] Documentation is updated/provided or not needed
- [x] Changes are tested locally

### CODE REVIEW

**Review by Pulya10c — APPROVED** [review]

File: client/src/pages/course/admin/stage-interviews.tsx [hunk 1]
```diff
@@ -46,17 +46,22 @@ function Page(props: CoursePageProps) {
...
-        </Button>
         {courseManagerRole ? (
           <div>
             <Checkbox checked={noRegistration} onChange={e => setNoRegistration(e.target.checked)}>
               No Registration
             </Checkbox>
-            <Button onClick={createInterviews}>Create Interviews</Button>
+            <Popconfirm
+              onConfirm={() => createInterviews()}
+              title="Do you want to create interview pairs for not distribute students?"
```
Comment (I-vasilich-I) [inline client/src/pages/course/admin/stage-interviews.tsx:56]: is it about not distributed yet students? then you should say `distributed`

File: client/src/pages/course/admin/stage-interviews.tsx [hunk 2]
```diff
@@ -46,17 +46,22 @@ function Page(props: CoursePageProps) {
...
+            >
+              <Button onClick={createInterviews}>Create Interview Pairs</Button>
+            </Popconfirm>
```
Comment (AlreadyBored) [inline client/src/pages/course/admin/stage-interviews.tsx:59]: Looks like a duplication. Maybe `okText` = `Create Interview Pairs` + remove `Button` child?

File: client/src/pages/course/admin/stage-interviews.tsx (hunk 1 above)
Comment (valerydluski) [reply client/src/pages/course/admin/stage-interviews.tsx]: fixed

**Review by AlreadyBored — APPROVED** [review]

File: client/src/pages/course/admin/stage-interviews.tsx (hunk 2 above)
Comment (valerydluski) [reply client/src/pages/course/admin/stage-interviews.tsx]: fixed

**Review by I-vasilich-I — APPROVED** [review]

**Pull request merged**